### ISSUE TITLE
Sleigh instruction with conditional prefix

### ISSUE THREAD

**Issue by astrelsky** [NationalSecurityAgency/ghidra#1183]
I'm looking to create a set of instructions with the condition prefix "v". (vadd, vsub, vmul etc) I setup a table called VPREFIX and placed it in front of the mnemonic surrounded in quotes and separated by ^. ex : `VPREFIX^"add"` Unfortunately this causes sleigh to think that every instruction has the same mnemonic and eventually runs out of heap space during compilation. Is there a way I may do this without having to define vadd and add separately when they do the same thing but have a different mnemonic depending on context flags?

**Comment by mumbel** [comment]
try `:^VPREFIX^"add"`

**Comment by astrelsky** [comment]
> try `:^VPREFIX^"add"`

That was the first thing I tried. I keep running out of heap space during compilation which I think is because sleigh is dumping them all into one table and trying to determine which one to use from the patter fields. Basically the mnemonic for all those instructions becomes "".

I believe the correct was is to define them without the prefix and use | in the pattern then do the following.
```
:VPREFIX^instruction is context {};
:instruction is epsilon {};
```

When I tried adding both conditions for the instruction (with and without the context register set) and also ran out of heap space. I think I just have an instruction causing trouble somewhere that I need to track down. I'll have to come back to this.
